### PR TITLE
Improve E-puck Model

### PR DESCRIPTION
**Pull request by DavidMansolino**
This PR adds various improvements to the e-puck robot model (especially for the v2) and adds the pi-puck extension.

The robot with the pi-puck extension:

e-puck2

Some more ideas to improve the model:
  - [x] Improve the led cylinder by using `SolidPipe` with a bigger subdivision.
  - [x] Make the led of the led ring smaller and with a higher intensity.
  - [x] Make a model of the battery V2 (rounded + 2 colors).
  - [x] Add the battery connectors.
  - [x] Improve meshes
  - [x] Create PBR textures.
  - [x] Top USB as mesh.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
With fixed body meshes + PBR textures + improved LEDs + USB plug + improved battery mesh (+many more):

e-puck2_2
e-puck2_3

**Comment by DavidMansolino** [comment]
Some of the improvments are also available for the epuck v1:

| revision | this branch |
| :---: | :---: |
| e-puck_cross-compilation_1 | e-puck_cross-compilation_2 |

**Comment by DavidMansolino** [comment]
:+1: :heavy_check_mark:

File: projects/robots/gctronic/e-puck/worlds/.e-puck2_pi-puck.wbproj [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Webots Project File version R2020a
+perspectives: 000000ff00000000fd00000003000000000000000000000000fc0100000001fb0000001a0044006f00630075006d0065006e0074006100740069006f006e0000000000000000140000006900ffffff000000010000011c00000375fc0200000001fb0000001400540065007800740045006400690074006f00720100000000000003750000003f00ffffff000000030000073f00000093fc0100000001fb0000000e0043006f006e0073006f006c006501000000000000073f0000006900ffffff000006210000037500000001000000020000000100000008fc00000000
+simulationViewPerspectives: 000000ff000000010000000200000100000005210100000002010000000100
+sceneTreePerspectives: 000000ff0000000100000002000002aa000000fa0100000002010000000200
+maximizedDockId: -1
+centralWidgetVisible: 1
+selectionDisabled: 0
+viewpointLocked: 0
+orthographicViewHeight: 1
+textFiles: -1
```
Comment (fabienrohrer) [inline projects/robots/gctronic/e-puck/worlds/.e-puck2_pi-puck.wbproj]: Wouldn't be good to open the controller file there?

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**